sources/web/datalab/polymer/test/files-test.html

```diff
@@ -0,0 +1,41 @@
+<!--
+Copyright 2017 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+in compliance with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License
+is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+or implied. See the License for the specific language governing permissions and limitations under
+the License.
+-->
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+    <script src="../../web-component-tester/browser.js"></script>
+    <script src="../modules/Utils.js"></script>
+    <script src="../modules/ApiManager.js"></script>
+    <script src="../modules/SettingsManager.js"></script>
+
+    <link rel="import" href="../components/datalab-files/datalab-files.html">
+  </head>
+  <body>
+
+    <test-fixture id="files-fixture">
+      <template>
+        <datalab-files></datalab-files>
+      </template>
+    </test-fixture>
+
+    <script src="files-test.js"></script>
```

Comment:
All the other test files so far are named <file-under-test>-test.{html,ts}. Following that pattern, these files should be named datalab-files-test.html and datalab-files-test.ts. Or you could rename datalab-files.html and datalab-files.ts to files.html and files.ts.